lib/winrm/http/transport.rb

```diff
@@ -173,13 +253,13 @@ def send_kerberos_request(message)
             'boundary="Encrypted Boundary"'
         }
 
-        body = <<-EOF
---Encrypted Boundary\r
-Content-Type: application/HTTP-Kerberos-session-encrypted\r
-OriginalContent: type=application/soap+xml;charset=UTF-8;Length=#{original_length + pad_len}\r
---Encrypted Boundary\r
-Content-Type: application/octet-stream\r
-#{emsg}--Encrypted Boundary\r
+        body = <<-EOF.gsub(/^\s{10}/,"")
```

Comment:
I agree that unindented heredocs are ugly, but I think `gsub`'ing indentation off is only marginally better. I usually prefer a construct like this:

```ruby
body = [
    "--Encrypted Boundary",
    "Content-Type: application/HTTP-Kerberos-session-encrypted",
    ...
].join("\r\n")
```
which also has the advantage of surviving a unix2dos(1) conversion.